cmd/lncli/commands.go

```diff
@@ -3173,10 +3174,36 @@ var forwardingHistoryCommand = cli.Command{
 			Name:  "max_events",
 			Usage: "the max number of events to return",
 		},
+		cli.StringFlag{
+			Name:  "since",
+			Usage: "number of H(ours)/D(ays)/W(eeks) before end_time (if negative) or after start_time or current time if no start_time specifued",
+		},		
 	},
 	Action: actionDecorator(forwardingHistory),
 }
 
+func translateSince(sinceVal string) (ret int64,  err error) {
+	//convert first length-1 characters to int
+	count, err := strconv.Atoi(sinceVal[0:len(sinceVal)-1])
+	if err != nil {
+		ret = int64(0)
+	} else  {	
+	
+	unit := sinceVal[len(sinceVal)-1:len(sinceVal)] 	
+	switch {
+		case unit == "H" :
+			ret = int64(3600*count)
+		case unit == "D" :
+		    ret = int64(24*3600*count)
+		case unit == "W" :
+			ret = int64(7*24*3600*count)  
+		default:
+		    err = errors.New("Time unit must be H,D or W")	  	
+	}
+	}
```

Comment:
You should run gofmt before commiting. It makes worrying about code formatting a non-issue.
There are many tools (e.g. go-plus for Atom), that do this automatically for you.

Reply:
I will do this. Sorry, I am sure that was in the instructions for contributors.